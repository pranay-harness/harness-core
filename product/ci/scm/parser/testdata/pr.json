{
    "action": "opened",
    "number": 1,
    "pull_request": {
        "url": "https://api.github.com/repos/wings-software/test-webhook/pulls/1",
        "id": 470162619,
        "node_id": "MDExOlB1bGxSZXF1ZXN0NDcwMTYyNjE5",
        "html_url": "https://github.com/wings-software/test-webhook/pull/1",
        "diff_url": "https://github.com/wings-software/test-webhook/pull/1.diff",
        "patch_url": "https://github.com/wings-software/test-webhook/pull/1.patch",
        "issue_url": "https://api.github.com/repos/wings-software/test-webhook/issues/1",
        "number": 1,
        "state": "open",
        "locked": false,
        "title": "Update README.md",
        "user": {
            "login": "shubham149",
            "id": 61136843,
            "node_id": "MDQ6VXNlcjYxMTM2ODQz",
            "avatar_url": "https://avatars1.githubusercontent.com/u/61136843?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/shubham149",
            "html_url": "https://github.com/shubham149",
            "followers_url": "https://api.github.com/users/shubham149/followers",
            "following_url": "https://api.github.com/users/shubham149/following{/other_user}",
            "gists_url": "https://api.github.com/users/shubham149/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/shubham149/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/shubham149/subscriptions",
            "organizations_url": "https://api.github.com/users/shubham149/orgs",
            "repos_url": "https://api.github.com/users/shubham149/repos",
            "events_url": "https://api.github.com/users/shubham149/events{/privacy}",
            "received_events_url": "https://api.github.com/users/shubham149/received_events",
            "type": "User",
            "site_admin": false
        },
        "body": "",
        "created_at": "2020-08-19T13:24:14Z",
        "updated_at": "2020-08-19T13:24:14Z",
        "closed_at": null,
        "merged_at": null,
        "merge_commit_sha": null,
        "assignee": null,
        "assignees": [],
        "requested_reviewers": [],
        "requested_teams": [],
        "labels": [],
        "milestone": null,
        "draft": false,
        "commits_url": "https://api.github.com/repos/wings-software/test-webhook/pulls/1/commits",
        "review_comments_url": "https://api.github.com/repos/wings-software/test-webhook/pulls/1/comments",
        "review_comment_url": "https://api.github.com/repos/wings-software/test-webhook/pulls/comments{/number}",
        "comments_url": "https://api.github.com/repos/wings-software/test-webhook/issues/1/comments",
        "statuses_url": "https://api.github.com/repos/wings-software/test-webhook/statuses/69c3de0ed28d05d92bb57d685f23bc64934fb80d",
        "head": {
            "label": "wings-software:shubham149-patch-1",
            "ref": "shubham149-patch-1",
            "sha": "69c3de0ed28d05d92bb57d685f23bc64934fb80d",
            "user": {
                "login": "wings-software",
                "id": 18273000,
                "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
                "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/wings-software",
                "html_url": "https://github.com/wings-software",
                "followers_url": "https://api.github.com/users/wings-software/followers",
                "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
                "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
                "organizations_url": "https://api.github.com/users/wings-software/orgs",
                "repos_url": "https://api.github.com/users/wings-software/repos",
                "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
                "received_events_url": "https://api.github.com/users/wings-software/received_events",
                "type": "Organization",
                "site_admin": false
            },
            "repo": {
                "id": 288719292,
                "node_id": "MDEwOlJlcG9zaXRvcnkyODg3MTkyOTI=",
                "name": "test-webhook",
                "full_name": "wings-software/test-webhook",
                "private": true,
                "owner": {
                    "login": "wings-software",
                    "id": 18273000,
                    "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
                    "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/wings-software",
                    "html_url": "https://github.com/wings-software",
                    "followers_url": "https://api.github.com/users/wings-software/followers",
                    "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
                    "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
                    "organizations_url": "https://api.github.com/users/wings-software/orgs",
                    "repos_url": "https://api.github.com/users/wings-software/repos",
                    "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/wings-software/received_events",
                    "type": "Organization",
                    "site_admin": false
                },
                "html_url": "https://github.com/wings-software/test-webhook",
                "description": null,
                "fork": false,
                "url": "https://api.github.com/repos/wings-software/test-webhook",
                "forks_url": "https://api.github.com/repos/wings-software/test-webhook/forks",
                "keys_url": "https://api.github.com/repos/wings-software/test-webhook/keys{/key_id}",
                "collaborators_url": "https://api.github.com/repos/wings-software/test-webhook/collaborators{/collaborator}",
                "teams_url": "https://api.github.com/repos/wings-software/test-webhook/teams",
                "hooks_url": "https://api.github.com/repos/wings-software/test-webhook/hooks",
                "issue_events_url": "https://api.github.com/repos/wings-software/test-webhook/issues/events{/number}",
                "events_url": "https://api.github.com/repos/wings-software/test-webhook/events",
                "assignees_url": "https://api.github.com/repos/wings-software/test-webhook/assignees{/user}",
                "branches_url": "https://api.github.com/repos/wings-software/test-webhook/branches{/branch}",
                "tags_url": "https://api.github.com/repos/wings-software/test-webhook/tags",
                "blobs_url": "https://api.github.com/repos/wings-software/test-webhook/git/blobs{/sha}",
                "git_tags_url": "https://api.github.com/repos/wings-software/test-webhook/git/tags{/sha}",
                "git_refs_url": "https://api.github.com/repos/wings-software/test-webhook/git/refs{/sha}",
                "trees_url": "https://api.github.com/repos/wings-software/test-webhook/git/trees{/sha}",
                "statuses_url": "https://api.github.com/repos/wings-software/test-webhook/statuses/{sha}",
                "languages_url": "https://api.github.com/repos/wings-software/test-webhook/languages",
                "stargazers_url": "https://api.github.com/repos/wings-software/test-webhook/stargazers",
                "contributors_url": "https://api.github.com/repos/wings-software/test-webhook/contributors",
                "subscribers_url": "https://api.github.com/repos/wings-software/test-webhook/subscribers",
                "subscription_url": "https://api.github.com/repos/wings-software/test-webhook/subscription",
                "commits_url": "https://api.github.com/repos/wings-software/test-webhook/commits{/sha}",
                "git_commits_url": "https://api.github.com/repos/wings-software/test-webhook/git/commits{/sha}",
                "comments_url": "https://api.github.com/repos/wings-software/test-webhook/comments{/number}",
                "issue_comment_url": "https://api.github.com/repos/wings-software/test-webhook/issues/comments{/number}",
                "contents_url": "https://api.github.com/repos/wings-software/test-webhook/contents/{+path}",
                "compare_url": "https://api.github.com/repos/wings-software/test-webhook/compare/{base}...{head}",
                "merges_url": "https://api.github.com/repos/wings-software/test-webhook/merges",
                "archive_url": "https://api.github.com/repos/wings-software/test-webhook/{archive_format}{/ref}",
                "downloads_url": "https://api.github.com/repos/wings-software/test-webhook/downloads",
                "issues_url": "https://api.github.com/repos/wings-software/test-webhook/issues{/number}",
                "pulls_url": "https://api.github.com/repos/wings-software/test-webhook/pulls{/number}",
                "milestones_url": "https://api.github.com/repos/wings-software/test-webhook/milestones{/number}",
                "notifications_url": "https://api.github.com/repos/wings-software/test-webhook/notifications{?since,all,participating}",
                "labels_url": "https://api.github.com/repos/wings-software/test-webhook/labels{/name}",
                "releases_url": "https://api.github.com/repos/wings-software/test-webhook/releases{/id}",
                "deployments_url": "https://api.github.com/repos/wings-software/test-webhook/deployments",
                "created_at": "2020-08-19T11:57:57Z",
                "updated_at": "2020-08-19T13:19:26Z",
                "pushed_at": "2020-08-19T13:24:09Z",
                "git_url": "git://github.com/wings-software/test-webhook.git",
                "ssh_url": "git@github.com:wings-software/test-webhook.git",
                "clone_url": "https://github.com/wings-software/test-webhook.git",
                "svn_url": "https://github.com/wings-software/test-webhook",
                "homepage": null,
                "size": 0,
                "stargazers_count": 0,
                "watchers_count": 0,
                "language": null,
                "has_issues": true,
                "has_projects": true,
                "has_downloads": true,
                "has_wiki": true,
                "has_pages": false,
                "forks_count": 0,
                "mirror_url": null,
                "archived": false,
                "disabled": false,
                "open_issues_count": 1,
                "license": null,
                "forks": 0,
                "open_issues": 1,
                "watchers": 0,
                "default_branch": "master",
                "allow_squash_merge": true,
                "allow_merge_commit": true,
                "allow_rebase_merge": true,
                "delete_branch_on_merge": false
            }
        },
        "base": {
            "label": "wings-software:master",
            "ref": "master",
            "sha": "e286f86f4db84e682b50007772a28a9d5e9b51d0",
            "user": {
                "login": "wings-software",
                "id": 18273000,
                "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
                "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/wings-software",
                "html_url": "https://github.com/wings-software",
                "followers_url": "https://api.github.com/users/wings-software/followers",
                "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
                "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
                "organizations_url": "https://api.github.com/users/wings-software/orgs",
                "repos_url": "https://api.github.com/users/wings-software/repos",
                "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
                "received_events_url": "https://api.github.com/users/wings-software/received_events",
                "type": "Organization",
                "site_admin": false
            },
            "repo": {
                "id": 288719292,
                "node_id": "MDEwOlJlcG9zaXRvcnkyODg3MTkyOTI=",
                "name": "test-webhook",
                "full_name": "wings-software/test-webhook",
                "private": true,
                "owner": {
                    "login": "wings-software",
                    "id": 18273000,
                    "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
                    "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
                    "gravatar_id": "",
                    "url": "https://api.github.com/users/wings-software",
                    "html_url": "https://github.com/wings-software",
                    "followers_url": "https://api.github.com/users/wings-software/followers",
                    "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
                    "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
                    "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
                    "organizations_url": "https://api.github.com/users/wings-software/orgs",
                    "repos_url": "https://api.github.com/users/wings-software/repos",
                    "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
                    "received_events_url": "https://api.github.com/users/wings-software/received_events",
                    "type": "Organization",
                    "site_admin": false
                },
                "html_url": "https://github.com/wings-software/test-webhook",
                "description": null,
                "fork": false,
                "url": "https://api.github.com/repos/wings-software/test-webhook",
                "forks_url": "https://api.github.com/repos/wings-software/test-webhook/forks",
                "keys_url": "https://api.github.com/repos/wings-software/test-webhook/keys{/key_id}",
                "collaborators_url": "https://api.github.com/repos/wings-software/test-webhook/collaborators{/collaborator}",
                "teams_url": "https://api.github.com/repos/wings-software/test-webhook/teams",
                "hooks_url": "https://api.github.com/repos/wings-software/test-webhook/hooks",
                "issue_events_url": "https://api.github.com/repos/wings-software/test-webhook/issues/events{/number}",
                "events_url": "https://api.github.com/repos/wings-software/test-webhook/events",
                "assignees_url": "https://api.github.com/repos/wings-software/test-webhook/assignees{/user}",
                "branches_url": "https://api.github.com/repos/wings-software/test-webhook/branches{/branch}",
                "tags_url": "https://api.github.com/repos/wings-software/test-webhook/tags",
                "blobs_url": "https://api.github.com/repos/wings-software/test-webhook/git/blobs{/sha}",
                "git_tags_url": "https://api.github.com/repos/wings-software/test-webhook/git/tags{/sha}",
                "git_refs_url": "https://api.github.com/repos/wings-software/test-webhook/git/refs{/sha}",
                "trees_url": "https://api.github.com/repos/wings-software/test-webhook/git/trees{/sha}",
                "statuses_url": "https://api.github.com/repos/wings-software/test-webhook/statuses/{sha}",
                "languages_url": "https://api.github.com/repos/wings-software/test-webhook/languages",
                "stargazers_url": "https://api.github.com/repos/wings-software/test-webhook/stargazers",
                "contributors_url": "https://api.github.com/repos/wings-software/test-webhook/contributors",
                "subscribers_url": "https://api.github.com/repos/wings-software/test-webhook/subscribers",
                "subscription_url": "https://api.github.com/repos/wings-software/test-webhook/subscription",
                "commits_url": "https://api.github.com/repos/wings-software/test-webhook/commits{/sha}",
                "git_commits_url": "https://api.github.com/repos/wings-software/test-webhook/git/commits{/sha}",
                "comments_url": "https://api.github.com/repos/wings-software/test-webhook/comments{/number}",
                "issue_comment_url": "https://api.github.com/repos/wings-software/test-webhook/issues/comments{/number}",
                "contents_url": "https://api.github.com/repos/wings-software/test-webhook/contents/{+path}",
                "compare_url": "https://api.github.com/repos/wings-software/test-webhook/compare/{base}...{head}",
                "merges_url": "https://api.github.com/repos/wings-software/test-webhook/merges",
                "archive_url": "https://api.github.com/repos/wings-software/test-webhook/{archive_format}{/ref}",
                "downloads_url": "https://api.github.com/repos/wings-software/test-webhook/downloads",
                "issues_url": "https://api.github.com/repos/wings-software/test-webhook/issues{/number}",
                "pulls_url": "https://api.github.com/repos/wings-software/test-webhook/pulls{/number}",
                "milestones_url": "https://api.github.com/repos/wings-software/test-webhook/milestones{/number}",
                "notifications_url": "https://api.github.com/repos/wings-software/test-webhook/notifications{?since,all,participating}",
                "labels_url": "https://api.github.com/repos/wings-software/test-webhook/labels{/name}",
                "releases_url": "https://api.github.com/repos/wings-software/test-webhook/releases{/id}",
                "deployments_url": "https://api.github.com/repos/wings-software/test-webhook/deployments",
                "created_at": "2020-08-19T11:57:57Z",
                "updated_at": "2020-08-19T13:19:26Z",
                "pushed_at": "2020-08-19T13:24:09Z",
                "git_url": "git://github.com/wings-software/test-webhook.git",
                "ssh_url": "git@github.com:wings-software/test-webhook.git",
                "clone_url": "https://github.com/wings-software/test-webhook.git",
                "svn_url": "https://github.com/wings-software/test-webhook",
                "homepage": null,
                "size": 0,
                "stargazers_count": 0,
                "watchers_count": 0,
                "language": null,
                "has_issues": true,
                "has_projects": true,
                "has_downloads": true,
                "has_wiki": true,
                "has_pages": false,
                "forks_count": 0,
                "mirror_url": null,
                "archived": false,
                "disabled": false,
                "open_issues_count": 1,
                "license": null,
                "forks": 0,
                "open_issues": 1,
                "watchers": 0,
                "default_branch": "master",
                "allow_squash_merge": true,
                "allow_merge_commit": true,
                "allow_rebase_merge": true,
                "delete_branch_on_merge": false
            }
        },
        "_links": {
            "self": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/pulls/1"
            },
            "html": {
                "href": "https://github.com/wings-software/test-webhook/pull/1"
            },
            "issue": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/issues/1"
            },
            "comments": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/issues/1/comments"
            },
            "review_comments": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/pulls/1/comments"
            },
            "review_comment": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/pulls/comments{/number}"
            },
            "commits": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/pulls/1/commits"
            },
            "statuses": {
                "href": "https://api.github.com/repos/wings-software/test-webhook/statuses/69c3de0ed28d05d92bb57d685f23bc64934fb80d"
            }
        },
        "author_association": "COLLABORATOR",
        "active_lock_reason": null,
        "merged": false,
        "mergeable": null,
        "rebaseable": null,
        "mergeable_state": "unknown",
        "merged_by": null,
        "comments": 0,
        "review_comments": 0,
        "maintainer_can_modify": false,
        "commits": 1,
        "additions": 1,
        "deletions": 0,
        "changed_files": 1
    },
    "repository": {
        "id": 288719292,
        "node_id": "MDEwOlJlcG9zaXRvcnkyODg3MTkyOTI=",
        "name": "test-webhook",
        "full_name": "wings-software/test-webhook",
        "private": true,
        "owner": {
            "login": "wings-software",
            "id": 18273000,
            "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
            "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/wings-software",
            "html_url": "https://github.com/wings-software",
            "followers_url": "https://api.github.com/users/wings-software/followers",
            "following_url": "https://api.github.com/users/wings-software/following{/other_user}",
            "gists_url": "https://api.github.com/users/wings-software/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/wings-software/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/wings-software/subscriptions",
            "organizations_url": "https://api.github.com/users/wings-software/orgs",
            "repos_url": "https://api.github.com/users/wings-software/repos",
            "events_url": "https://api.github.com/users/wings-software/events{/privacy}",
            "received_events_url": "https://api.github.com/users/wings-software/received_events",
            "type": "Organization",
            "site_admin": false
        },
        "html_url": "https://github.com/wings-software/test-webhook",
        "description": null,
        "fork": false,
        "url": "https://api.github.com/repos/wings-software/test-webhook",
        "forks_url": "https://api.github.com/repos/wings-software/test-webhook/forks",
        "keys_url": "https://api.github.com/repos/wings-software/test-webhook/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/wings-software/test-webhook/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/wings-software/test-webhook/teams",
        "hooks_url": "https://api.github.com/repos/wings-software/test-webhook/hooks",
        "issue_events_url": "https://api.github.com/repos/wings-software/test-webhook/issues/events{/number}",
        "events_url": "https://api.github.com/repos/wings-software/test-webhook/events",
        "assignees_url": "https://api.github.com/repos/wings-software/test-webhook/assignees{/user}",
        "branches_url": "https://api.github.com/repos/wings-software/test-webhook/branches{/branch}",
        "tags_url": "https://api.github.com/repos/wings-software/test-webhook/tags",
        "blobs_url": "https://api.github.com/repos/wings-software/test-webhook/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/wings-software/test-webhook/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/wings-software/test-webhook/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/wings-software/test-webhook/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/wings-software/test-webhook/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/wings-software/test-webhook/languages",
        "stargazers_url": "https://api.github.com/repos/wings-software/test-webhook/stargazers",
        "contributors_url": "https://api.github.com/repos/wings-software/test-webhook/contributors",
        "subscribers_url": "https://api.github.com/repos/wings-software/test-webhook/subscribers",
        "subscription_url": "https://api.github.com/repos/wings-software/test-webhook/subscription",
        "commits_url": "https://api.github.com/repos/wings-software/test-webhook/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/wings-software/test-webhook/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/wings-software/test-webhook/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/wings-software/test-webhook/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/wings-software/test-webhook/contents/{+path}",
        "compare_url": "https://api.github.com/repos/wings-software/test-webhook/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/wings-software/test-webhook/merges",
        "archive_url": "https://api.github.com/repos/wings-software/test-webhook/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/wings-software/test-webhook/downloads",
        "issues_url": "https://api.github.com/repos/wings-software/test-webhook/issues{/number}",
        "pulls_url": "https://api.github.com/repos/wings-software/test-webhook/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/wings-software/test-webhook/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/wings-software/test-webhook/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/wings-software/test-webhook/labels{/name}",
        "releases_url": "https://api.github.com/repos/wings-software/test-webhook/releases{/id}",
        "deployments_url": "https://api.github.com/repos/wings-software/test-webhook/deployments",
        "created_at": "2020-08-19T11:57:57Z",
        "updated_at": "2020-08-19T13:19:26Z",
        "pushed_at": "2020-08-19T13:24:09Z",
        "git_url": "git://github.com/wings-software/test-webhook.git",
        "ssh_url": "git@github.com:wings-software/test-webhook.git",
        "clone_url": "https://github.com/wings-software/test-webhook.git",
        "svn_url": "https://github.com/wings-software/test-webhook",
        "homepage": null,
        "size": 0,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": null,
        "has_issues": true,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": true,
        "has_pages": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 1,
        "license": null,
        "forks": 0,
        "open_issues": 1,
        "watchers": 0,
        "default_branch": "master"
    },
    "organization": {
        "login": "wings-software",
        "id": 18273000,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjE4MjczMDAw",
        "url": "https://api.github.com/orgs/wings-software",
        "repos_url": "https://api.github.com/orgs/wings-software/repos",
        "events_url": "https://api.github.com/orgs/wings-software/events",
        "hooks_url": "https://api.github.com/orgs/wings-software/hooks",
        "issues_url": "https://api.github.com/orgs/wings-software/issues",
        "members_url": "https://api.github.com/orgs/wings-software/members{/member}",
        "public_members_url": "https://api.github.com/orgs/wings-software/public_members{/member}",
        "avatar_url": "https://avatars1.githubusercontent.com/u/18273000?v=4",
        "description": ""
    },
    "sender": {
        "login": "shubham149",
        "id": 61136843,
        "node_id": "MDQ6VXNlcjYxMTM2ODQz",
        "avatar_url": "https://avatars1.githubusercontent.com/u/61136843?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/shubham149",
        "html_url": "https://github.com/shubham149",
        "followers_url": "https://api.github.com/users/shubham149/followers",
        "following_url": "https://api.github.com/users/shubham149/following{/other_user}",
        "gists_url": "https://api.github.com/users/shubham149/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/shubham149/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/shubham149/subscriptions",
        "organizations_url": "https://api.github.com/users/shubham149/orgs",
        "repos_url": "https://api.github.com/users/shubham149/repos",
        "events_url": "https://api.github.com/users/shubham149/events{/privacy}",
        "received_events_url": "https://api.github.com/users/shubham149/received_events",
        "type": "User",
        "site_admin": false
    }
}