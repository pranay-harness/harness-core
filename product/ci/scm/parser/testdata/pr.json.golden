{
    "pr": {
        "action": "OPEN",
        "repo": {
            "id": "288719292",
            "namespace": "wings-software",
            "name": "test-webhook",
            "branch": "master",
            "private": true,
            "clone": "https://github.com/wings-software/test-webhook.git",
            "cloneSsh": "git@github.com:wings-software/test-webhook.git",
            "link": "https://github.com/wings-software/test-webhook",
            "created": "0001-01-01T00:00:00Z",
            "updated": "0001-01-01T00:00:00Z"
        },
        "pr": {
            "number": "2",
            "title": "Update README.md",
            "body": "pr request",
            "sha": "616993848535117531eea23f39aa046999f5237f",
            "ref": "refs/pull/2/head",
            "source": "shubham149-patch-2",
            "target": "master",
            "fork": "wings-software/test-webhook",
            "link": "https://github.com/wings-software/test-webhook/pull/2",
            "base": {
                "name": "master",
                "path": "refs/heads/master",
                "sha": "bafbacb95e8941fdc1cfced3d229bd15ddc2bc8b"
            },
            "head": {
                "name": "shubham149-patch-2",
                "path": "refs/heads/shubham149-patch-2",
                "sha": "616993848535117531eea23f39aa046999f5237f"
            },
            "author": {
                "login": "shubham149",
                "avatar": "https://avatars1.githubusercontent.com/u/61136843?v=4",
                "created": "0001-01-01T00:00:00Z",
                "updated": "0001-01-01T00:00:00Z"
            },
            "created": "2020-09-03T09:25:26Z",
            "updated": "2020-09-03T09:25:26Z",
            "labels": [
                {
                    "name": "go",
                    "color": "f4a869"
                },
                {
                    "name": "good first issue",
                    "color": "7057ff"
                }
            ]
        },
        "sender": {
            "login": "shubham149",
            "avatar": "https://avatars1.githubusercontent.com/u/61136843?v=4",
            "created": "0001-01-01T00:00:00Z",
            "updated": "0001-01-01T00:00:00Z"
        }
    }
}